src/Tmds.Ssh/PrivateKeyCredential.OpenSsh.cs

```diff
@@ -0,0 +1,272 @@
+// This file is part of Tmds.Ssh which is released under MIT.
+// See file LICENSE for full license details.
+
+using System;
+using System.Buffers;
+using System.Diagnostics.CodeAnalysis;
+using System.Numerics;
+using System.Security.Cryptography;
+
+namespace Tmds.Ssh;
+
+partial class PrivateKeyCredential
+{
+    internal static bool TryParseOpenSshKey(
```

Comment:
Can you move this to `PublicKeyAuth` or under a new helper class, like `PrivateKeyParser`.

Reply:
I went with the latter.